app/controllers/alchemy/admin/contents_controller.rb

```diff
@@ -32,9 +32,10 @@ def update
       end
 
       def order
-        params[:content_ids].each do |id|
-          content = Content.find(id)
-          content.move_to_bottom
+        Content.transaction do
+          params[:content_ids].each_with_index do |id, idx|
+            Content.where(id: id).update_all(position: idx + 1)
```

Comment:
`find_by(...).update_attributes(...)` instantiates the object and then runs the update. Two queries, and objects in the middle - we better stay with the current code. Ignore me. :)